src/main/resources/db/changelog/changes/v1.0.0/sql/entity-type-definitions/insert-user-details-definition.sql

```diff
@@ -154,7 +154,30 @@ INSERT INTO entity_type_definition (id, derived_table_name, definition)
                          "dataType":"stringType"
                        },
                        "labelAlias": "User preferred contact type",
-                       "visibleByDefault": false
+                       "visibleByDefault": false,
+                       "values": [
+                          {
+                            "value": "mail",
+                            "label": "Mail"
+                          },
+                          {
+                            "value": "email",
+                            "label": "Email"
+                          },
+                          {
+                            "value": "text",
+                            "label": "Text"
+                          },
+                          {
+                            "value": "phone",
+                            "label": "Phone"
+                          },
+                          {
+                            "value": "mobile",
+                            "label": "Mobile"
+                          }
+
+                      ]
```

Comment:
Need to add the drop down values like this since for preferred contact type does not come from db, they are hard coded values in the users-module

Reply:
At least it's easy